docs/sphinx/users/index.txt

```diff
@@ -121,6 +121,7 @@ Visualization and analysis applications
     imago/index
     iqm/index
     macnif/index
+    micromanager/index
```

Comment:
It might be worth discussing the categorization of Micro-Manager with @hflynn as it does not really fit into `visualization and analysis` but we have no other acquisition section.